website/www/site/content/en/contribute/contribute.md

```diff
@@ -0,0 +1,389 @@
+---
```

Comment:
As I can see you add `Beam Contribution Guide` page (rewrite from previous one). Make sure that [[BEAM-13992](https://issues.apache.org/jira/browse/BEAM-13992)] changes are also included

Reply:
added a comment to PR description to merge 13992 first and only after that 13991